### DISCUSSION TITLE
Custom Death Messagges

### DISCUSSION THREAD

**Discussion by Guest1131** [EssentialsX/Essentials#4158] [Ideas and Feature Suggestions] [answered]
Hello, I would be very happy if a feature was added in the essentials.

I saw that essentials, doesn't have any entry in the config, about deaths

For example in PVP, when a player kills another player with the sword, it would be written in chat a message such as "Player was killed by the Killer with the diamond sword".

**Comment by JRoy** [comment, marked as answer]
Bukkit does not expose any information to API on the death cause so this would be up to some 3rd party plugin to track.
